docs/versioned-plugins/filters/ruby-v3.1.1.asciidoc

```diff
@@ -86,7 +86,7 @@ def filter(event)
 end
 ----
 
-====== Testing the ruby script
+===== Testing the ruby script
```

Comment:
This problem is caused by bad asciidoc in older versions of the plugin. We can fix it with a simple search/replace in the old files.